test/e2e/run_networking_test.go

```diff
@@ -220,6 +220,22 @@ var _ = Describe("Podman run networking", func() {
 		Expect(inspectOut[0].NetworkSettings.Ports["8080/tcp"][0].HostIP).To(Equal(""))
 	})
 
+	It("podman run -p 8080:8080 -p 8081:8080", func() {
+		name := "testctr"
+		session := podmanTest.Podman([]string{"create", "-t", "-p", "8080:8080", "-p", "8081:8080", "--name", name, ALPINE, "/bin/sh"})
```

Comment:
Just to make it easier to figure out in the tests where things are coming from, could you change this to;
```suggestion
		session := podmanTest.Podman([]string{"create", "-t", "-p", "8082:8080", "-p", "8081:8080", "--name", name, ALPINE, "/bin/sh"})
```

and then adjust below as appropriate.

Reply:
Fixed